data/sql/updates/pending_db_world/rev_1641437831789184769.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1641437831789184769');
+
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2362200 AND `source_type`=9 AND `id`=2 AND `link`=0;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=23616 AND `source_type`=0 AND `id`=3 AND `link`=0;
```

Comment:
what is this deleting? id 3 line does not exsist in our db in the smart scripts

Reply:
Yeah is the requested Delete before Insert.